packages/google_maps_flutter/google_maps_flutter_web/lib/src/convert.dart

```diff
@@ -139,10 +139,11 @@ List<gmaps.MapTypeStyle> _mapStyles(String? mapStyleJson) {
       styles =
           (json.decode(mapStyleJson, reviver: (Object? key, Object? value) {
         if (value is Map && _isJsonMapStyle(value as Map<String, Object?>)) {
-          List<Object?> stylers = <Object?>[];
+          List<MapStyler> stylers = <MapStyler>[];
           if (value['stylers'] != null) {
             stylers = (value['stylers']! as List<Object?>)
-                .map<Object?>((Object? e) => e != null ? jsify(e) : null)
+                .whereType<Map<String, Object?>>()
```

Comment:
In the old version, we did pass `null`s to the styler objects?

Reply:
Tested with the demo app's "night mode" and this seems to work. There's also some tests incoming for this feature in https://github.com/flutter/packages/pull/6205